.travis.yml

```diff
@@ -19,19 +14,18 @@ matrix:
       env: TOXENV=quality
     - python: 3.5
       env: TOXENV=docs
-  exclude:
-    - python: 2.7
-      env: TOXENV=django20
-    - python: 2.7
-      env: TOXENV=django21
-    - python: 2.7
-      env: TOXENV=django22
+    - python: 3.8
```

Comment:
We can remove quality and docs  out of the matrix configuration and add that to the env configuration.